common/src/EL/Value.cpp

```diff
@@ -127,11 +130,11 @@ namespace TrenchBroom {
         ValueHolder* StringHolder::convertTo(const ValueType toType) const {
             switch (toType) {
                 case ValueType::Boolean:
-                    return new BooleanValueHolder(!StringUtils::caseSensitiveEqual(doGetValue(), "false") && !doGetValue().empty());
+                    return new BooleanValueHolder(!kdl::cs::is_equal(doGetValue(), "false") && !doGetValue().empty());
```

Comment:
I guess all of the `cs::` is string functions but it'd be a bit nicer to have `kdl::cs::str_is_equal`

Reply:
done